forge-gui/res/cardsfolder/g/glorious_gale.txt

```diff
@@ -0,0 +1,7 @@
+Name:Glorious Gale
+ManaCost:1 U
+Types:Instant
+A:SP$ Counter | Cost$ 1 U | TargetType$ Spell | TgtPrompt$ Select target creature spell | RememberCountered$ True | ValidTgts$ Card.Creature | SubAbility$ TrigTempt | SpellDescription$ Counter target creature spell. If it was a legendary spell, the Ring tempts you.
+SVar:TrigTempt:DB$ RingTemptsYou | ConditionDefined$ Remembered | ConditionPresent$ Card.Legendary | SubAbility$ DBCleanup

```

Comment:
that change seems wrong
even if the spell can't be countered you get the secondary effect

Reply:
does targetedLKI works?

Reply:
`ConditionDefined$ Targeted | ConditionPresent$ Spell.Legendary`